cmd/create/cluster/cmd.go

```diff
@@ -1682,8 +1682,13 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 
 	// Default machine pool labels
+	if isHostedCP && cmd.Flags().Changed("default-mp-labels") {
```

Comment:
NP
I think we can block the user earlier when we determine the value of `isHostedCP` in line `629`.

Reply:
fixed